tfhe/src/conformance.rs

```diff
@@ -60,8 +60,12 @@ impl ListSizeConstraint {
     }
 
     pub fn is_valid(&self, size: usize) -> bool {
-        size % self.group_size == 0
-            && size >= self.min_inclusive_group_count * self.group_size
-            && size <= self.max_inclusive_group_count * self.group_size
+        if self.group_size != 0 {
+            size % self.group_size == 0
+                && size >= self.min_inclusive_group_count * self.group_size
+                && size <= self.max_inclusive_group_count * self.group_size
+        } else {
+            size == 0
+        }
```

Comment:
self.group_size == 0 means only a size of 0 is accepted ? correct ?

as a general rule (and I think clippy might be saying the same) prefer positive predicates so

if self.group_size == 0 {
    size == 0
} else {
...
}